### DISCUSSION TITLE
Template customisation for lessons header

### DISCUSSION THREAD

**Discussion by mblue9** [carpentries/workbench#43] [Tips and Tricks]
In case this is useful for anyone else, this is an example of customising a lesson header
https://carpentries-incubator.github.io/bioc-rnaseq/

**Comment by zkamvar** [comment]
Hint for those unfamiliar: scroll to the bottom of the page and click on the {varnish} link. That will take you to @mblue9's fork of {varnish}, which modifies the HTML.
